package.json

```diff
@@ -697,7 +697,7 @@
                 },
                 {
                     "command": "cosmosDB.attachEmulator",
-                    "when": "view == azureDatabasesExplorer && viewItem == cosmosDBAttachedAccountsWithEmulator"
+                    "when": "view == azureDatabasesExplorer && viewItem =~ /^cosmosDBAttachedAccounts/"
```

Comment:
We only want to show the "Attach Emulator..." menu item if the context value is exactly `cosmosDBAttachedAccountsWithEmulator`. This regular expression shows that menu item for context values including `cosmosDBAttachedAccountsWithoutEmulator`

Reply:
That makes sense, I forgot to change it back to the original value.